tests/cli/test_compose_sanity.sh

```diff
@@ -42,15 +42,22 @@ rlJournalStart
         else
             rlFail "Compose UUID is empty!"
         fi
+
+        # check if anaconda is really running
+	until ps aux | egrep '[a]naconda'; do
```

Comment:
Why `[a]naconda` instead of just `grep anaconda` ?

Reply:
If I do just `ps aux | grep anaconda` it will always return `0` because it will find the grep command there running too. If I do it like `ps aux | grep '[a]naconda'` it will return `0` only if anaconda is running there, it won't find itself running.

Reply:
`ps -axo comm,pid | grep '^anaconda'` - ps with custom format and use grep to match at the beginning of the string. Alternatively you can also `grep -v grep`.
.

Reply:
Ok, that looks pretty neat. I found a post where someone was using `grep -v grep` but din't find it very nice and the `grep '[a]naconda' was suggested there and I kinda liked it so that's why I used it.